aviary/subsystems/mass/flops_based/landing_gear.py

```diff
@@ -250,6 +250,10 @@ def compute_partials(self, inputs, J):
 
 
 class NoseGearLength(om.ExplicitComponent):
+    """
+    Computation of landing gear OLEO length from main gear OLEO length:\n
+    NOSE_GEAR_OLEO_LENGTH = 0.7 * MAIN_GEAR_OLEO_LENGTH
```

Comment:
```suggestion
    Computation of nose gear oleo strut length from main gear oleo strut length:
    NOSE_GEAR_OLEO_LENGTH = 0.7 * MAIN_GEAR_OLEO_LENGTH
```
Oleo is just the name for a specific kind of shock absorbing strut in landing gear, see **https://en.wikipedia.org/wiki/Oleo_strut**